oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 partner-contact https://github.com/OCA/partner-contact 8.0
+intrastat https://github.com/sunflowerit/intrastat 8.0-intrastat_base-tests
```

Comment:
it was probably not merged at the time of the PR, so this was necessary to get the tests right. But by now, this should indeed just be 'intrastat'